src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2013,51 +2019,78 @@
       "description": "The analysis tool that was run.",
       "additionalProperties": false,
       "type": "object",
+      "properties": {
+
+        "language": {
+          "description": "The tool language (expressed as an ISO 649 two-letter lowercase culture code) and region (expressed as an ISO 3166 two-letter uppercase subculture code associated with a country or region).",
+          "type": "string",
+          "default": "en-US"
+        },
+
+        "driver": {
+          "description": "The analysis tool that was run.",
+          "$ref": "#/definitions/toolComponent"
+        },
+
+        "extensions": {
+          "description": "Tool extensions that contributed to or reconfigured the analysis tool run.",
```

Comment:
>run [](start = 96, length = 3)

"run" => "that was run" (for parallelism with description of "driver", and also I think it's clearer).